README.md

```diff
@@ -110,23 +126,64 @@ A simple way to define new handlers at the use site. Allows the use of positiona
     // .. which can then be called with arguments
     ~(embolden {"World"})
 
+### `if` *test* *trueBlock* *falseBlock*
+
+Conditional control flow.
+
+    let x = $(if true {4} {5}); // x == 4
```

Comment:
So, the test for `if` needs to be in a group just like the arms. Also relevant that the test will get expanded and then run, then only one of the arms will, with the other not getting run or expanded. So you can use `if` to guard against things that might explode in the expansion phase. Also might be nice to have a logic test shown in the if. The relevant section of the basic if test is included inline below for ref:
```rust   
      $(let
        x = {3}
        y = {4})

      $(if {$(logic $x < $y)} {let a = $y * 5;} {assert_eq!(4, $y);})
      $(if {$(logic $x > $y)} {let b = $y * 5;} {assert_eq!(4, $y);})
      $(if {true}  {assert_eq!(true, true);}  {assert_eq!(true, false);})
      $(if {false} {assert_eq!(true, false);} {assert_eq!(true, true);})
  };
  assert_eq!(a, 20);
```

Though I should probably change that test, now that I think about it, to something more like

```rust   
      $(let
        x = {3}
        y = {4})

      $(if {$(logic $x < $y)} {let a = $y * 5;} {assert_eq!(5, $y);})
      $(if {$(logic $x > $y)} {let a = $y * 7;} {assert_eq!(4, $y);})
      $(if {true}  {assert_eq!(true, true);}  {assert_eq!(true, false);})
      $(if {false} {assert_eq!(true, false);} {assert_eq!(true, true);})
  };
  assert_eq!(a, 20);
```

Hmm.